charm/jmx-exporter/metadata.yaml

```diff
@@ -3,9 +3,12 @@ summary: Exports JMX attributes to prometheus using jmx-exporter
 maintainer: Khanh Nguyen <khanh.nguyen@canonical.com>
 description: |
   JMX exporter charm
-min-juju-version: '2.4.0'
+min-juju-version: "2.4.0"
 series:
+  - xenial
```

Comment:
do we need to only support bionic for this?

Reply:
i don't think we need to support anything but focal and jammy.. same for charm (run-on section)